CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+Contributions are welcomed via [pull requests on GitHub](https://github.com/mosdef-hub/mbuild/pulls). Developers and/or 
+users will review requested changes and make comments. This The rest of this file will serve as a set of general guidelines
+for contributors.
+
+# Features
+
+## Implement functionality in a general and flexible fashion
+
+mBuild is designed to be general and flexible, not limited to single chemistries, file formats, simulations engines, or 
+simulation methods. Additions to core features shoudl attempt to provide something that is applicable to a variety of 
+use-cases and not  just the one thing you might need it to do for your research. But some specific features targetted toward 
+a limited use case may be appropriate. Speak to the developers before writing your code and they will help you make design 
+choices that allow flexibility.
+
+# Version control
+
+We currently use the "standard" Pull Request model. Contributions should be implemented on feature branches of forks.
+Please try to keep the `master` branch of your fork up-to-date with the `master` branch of the main repository.
```

Comment:
Not sure if appropriate to mention here or not, but I've added the following alias to my `.gitconfig` for convenience in achieving this:
`update = !git fetch upstream && git checkout master && git merge upstream/master`

Reply:
Oooh, that's nice. I'll pin this in slack